Source/VisualRecognitionV3/VisualRecognition.swift

```diff
@@ -941,6 +941,138 @@ public class VisualRecognition {
     }
  
     /**
+     Delete an image's metadata from a collection.
+     
+     - parameter withID: The ID of the collection to delete the image from.
+     - parameter imageID: The ID of the image to delete.
+     - parameter failure: A function executed if an error occurs.
+     - parameter success: A function executed when the image metadata is deleted successfully.
+    */
+    public func deleteImageMetadataFromCollection(
```

Comment:
Yeah, I had similar concerns. I ended up going with `deleteImageMetadataFromCollection` `withID` and `imageID` to clarify it's the ID rather than the name of the image/collection, and to follow previous function signatures that have similar features. At the same time, I like how `deleteImageMetadata` is concise.

I think I'll go with a mixture:

```swift
public func deleteImageMetadata(
    forImageID imageID: String,
    inCollectionID collectionID: String,
    ...
)
```

Let me know if you'd rather an alternative!